site/en/docs/privacy-sandbox/user-agent/index.md

```diff
@@ -113,6 +117,60 @@ If you need a specific set of Client Hints on your initial request, refer to
 [Client Hints Reliability](https://github.com/WICG/client-hints-infrastructure/blob/main/reliability.md)
 to ensure Client Hints are available on site load and optimized.
 
+## How do I prepare for reduced UA? {: #prepare-and-test}
+
+As we get closer to the rollout of the reduced User-Agent string in Chrome
+Stable, [review your site
+code](https://web.dev/migrate-to-ua-ch/#audit-collection-and-use-of-user-agent-data)
+for instances and uses of the User-Agent string. If your site relies on parsing
+the User-Agent string to read the device model, platform version, or full
+browser version, you’ll need to
+[implement the UA-CH API](https://web.dev/migrate-to-ua-ch/).
+
+Once you’ve updated to the UA-CH API, you should test to ensure you get the
+data you expect from the User-Agent. There are three ways to test, each
+increasing in complexity.
+
+### Test the string locally {: #test-locally}
+
+There are a couple of methods to test the reduced User-Agent locally:
+
+* Enable the `chrome://flags/#reduce-user-agent` flag.
+    * This will set your local browser to receive just the reduced `user-agent`
+      string for all sites, before it becomes the default setting.
+* Configure an emulated device in DevTools with the right `user-agent` string
+  and client hints.
+    * In DevTools under ⚙️ Settings → Devices → Add custom device...
```

Comment:
Style-guide question for @mihajlija. Do we have a convention for referring to settings locations? This comes from where I've done this in https://developer.chrome.com/docs/privacy-sandbox/user-agent/snippets/#local-testing-methods but strikes me that I may not have been consistent with other stuff. Specific questions:
- are the emoji valid here? I initially used them as DevTools has icons here.
- I think the individual items should be bold
- are we good with the arrows or do we have a convention for sub-menu selection?

Reply:
Re: the individual items & arrow conventions, I've updated to match the [GDSG multi-step format](https://developers.google.com/style/procedures#multi-action-procedures), which suggests `>` for moving between items and bolding them.

Reply:
I believe the emoji usage will be _okay_ in this instance, but would be much better as an inline image with alt text that matches the alt text in the UI.

Reply:
FYI emojis have been replaced with the same icons as the DevTools docs use

Reply:
Yeah, this works. Web.dev used to have components for this, but this is dcc, so we have to make do with workarounds.